docs/developers-clear-containers-install.md

```diff
@@ -124,13 +125,43 @@ See [the upgrading document](upgrading.md) for further details.
    $ sudo sed -i.bak -e 's!^\(image = ".*"\)!# \1 \
    image = "/usr/share/clear-containers/container.img"!g' \
    /usr/share/defaults/clear-containers/configuration.toml
-   
+   ```
 For more details on the runtime's build system, run:
 
 ```bash
 $ make help
 ```
 
+5. Kernel
+
+   The latest kernel for Clear Linux can be found in
```

Comment:
Lets embed this link:
...can be found on [the releases page](https://github.com/clearcontainers/linux/releases). The Clear Linux kernel can ...

Reply:
Added looks better !